Parsely/parsely-android
# Move `ParselyAPIConnection` to Kotlin

### Description

This small PR moves `ParselyAPIConnection` to Kotlin. I need it to be in Kotlin to introduce migrate from `AsyncTask` used in this class to Kotlin Coroutines. So please note, that this class will be heavily rewritten in next PRs.

### How to test

As this change changes code in HTTP communication, which is mocked in our tests, please:
- Update `MainActivity.java#31` and make the sample app not in the debug mode
- Tap on "track url"
- Move app to the background
- See in logs that POST request was successful
